Others/Dijkstra.java

```diff
@@ -56,7 +58,7 @@ public static void main(String[] args) {
         public final String v1, v2;
         public final int dist;

```

Comment:
I expected a `@NonNegative` annotation here, too. This field is initialized in the constructor with a non-negative value.

Reply:
Done!